### PR TITLE
Correct formatting of Possible Time Save Component

### PR DESCRIPTION
**Pull request by hallcristobal**
Corrected formatting of Possible Time Save Component

### CODE REVIEW

File: src/segment.rs [hunk 1]
```diff
@@ -59,6 +60,7 @@ impl Segment {
 
     #[inline]
     pub fn set_best_segment_time(&mut self, time: Time) {
+        self.comparisons.insert(BEST_SEGMENT_COMPARISON_NAME.into(), time);
```
Comment (CryZe) [inline src/segment.rs]: That's not what the C# code does. What's the purpose of this?

Comment (hallcristobal) [inline src/segment.rs]: The live split code adds the best segment to the comparisons via the best segments comparison generator, however that's not implemented yet but you are indexing comparisons on best segments which is returning None causing the live delta to not work correctly

Comment (CryZe) [inline src/segment.rs]: well, I'd rather have it not working fully atm, than introducing code that makes it temporarily work and cause bugs later on if we forget to remove it.

Comment (hallcristobal) [inline src/segment.rs]: that's fair enough, I'll remove it

**Pull request merged**